pypi_formula_mappings.json

```diff
@@ -19,6 +19,9 @@
     ],
     "exclude_packages": ["pycrypto", "six"]
   },
+  "ansible-lint": {
+    "exclude_packages": ["PyYAML", "jsonschema", "attrs", "pyrsistent", "yamllint", "pathspec", "ansible-core", "cryptography", "Jinja2", "MarkupSafe", "resolvelib"]
```

Comment:
I'm a little concerned about this list, since it's too easy for this to go stale.

Reply:
Maybe we should make `brew bump` smart enough to detect which Python modules are provided by dependent formulae, and automatically exclude those resources? Unfortunately, I don't know Ruby well enough to contribute that code myself, and I don't know how easy or hard that would be... it will probably require a standardized way of making one virtualenv depend on the virtualenv of a dependent package via a `.pth` file.

Reply:
I guess it can be surfaced out whenever we see in the version bump PR as well?